src/main/java/mate/academy/spring/dto/request/CinemaHallRequestDto.java

```diff
@@ -1,6 +1,9 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Size;
+
 public class CinemaHallRequestDto {
+    @Size(min = 1)
```

Comment:
Use Min for numbers
![image](https://user-images.githubusercontent.com/20326858/229487987-32617c73-950f-409c-8100-02869b835940.png)